Resource/PHPCSFixer/Fixer/OpeningTag.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Hoa\Devtools\Resource\PHPCsFixer\Fixer;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\FixerInterface;
+use Symfony\CS\Tokenizer\Tokens;
+use Symfony\CS\Tokenizer\Token;
+use SplFileInfo;
+
+/**
+ * Ensure there is a new line after the <php.
+ */
+class OpeningTag extends AbstractFixer
+{
+    public function fix(SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        if (!$tokens->isMonolithicPhp()) {
+            return $content;
+        }
+
+        if (!$tokens[0]->isGivenKind(T_OPEN_TAG)) {
+            return $content;
+        }
+
+        if (!$tokens[1]->isWhitespace()) {
+            $tokens[0]->setContent('<?php');
+            $tokens->insertAt(1, new Token([T_WHITESPACE, "\n\n"]));
```

Comment:
and the issue about your code is not that it is not optimized. the issue is that it does not set the open tag token to the same content than the one generated by PHP when parsing the file, which could create issues with other fixers expecting to have a token collection in a consistent state (the optimizations performed by the PHP-CS-Fixer mean that the same collection instance will be reused)